lib/powerdnsex/gen_server/server.ex

```diff
@@ -2,7 +2,11 @@ defmodule PowerDNSex.Server do
   use GenServer
 
   alias PowerDNSex.Managers.{ZonesManager, RecordsManager}
-  alias PowerDNSex.{Models.Zone, Converter}
+  alias PowerDNSex.{Converter}
```

Comment:
Moustaches not required: you are not aliasing more than one module:

```suggestion
  alias PowerDNSex.Converter
```